packages/card/newsroom-card/index.tsx

```diff
@@ -9,28 +9,32 @@ import type { CardProps, ThumbnailProps } from '../types'
 export interface NewsroomCardProps {
   heading: string
   date: string
+  category: string
   link: string
-  thumbnail: ThumbnailProps
+  thumbnail?: ThumbnailProps
   appearance?: CardProps['appearance']
+  withArrow?: boolean
 }
 
 export function NewsroomCard({
   heading,
   date,
+  category,
   link,
   thumbnail,
   appearance,
+  withArrow = false,
 }: NewsroomCardProps): JSX.Element {
   return (
     <CardPrimitives.Card
       heading={heading}
       link={link}
-      withArrow={false}
+      withArrow={withArrow}
       appearance={appearance}
     >
-      <CardPrimitives.Thumbnail {...thumbnail} />
+      {thumbnail ? <CardPrimitives.LogoThumbnail {...thumbnail} /> : null}
       <CardPrimitives.Content>
-        <CardPrimitives.Meta items={[date]} />
+        <CardPrimitives.Meta items={[date, category]} />
```

Comment:
Items in the newsroom come in one of three post types:
- Press releases
- Media coverage
- Analyst reports

To ensure all three are covered by `<NewsroomCard />`:
- I've made `withArrow` controllable via a top-level prop (for press releases, primarily)
- I've ensured the thumbnail component in use is styled for arbitrary company logos instead of `16:9` images.
- I've made sure the thumbnail is also conditionally rendered, so press releases can omit it and render only the eyebrow/heading/arrow